src/ar/akwam/src/eu/kanade/tachiyomi/animeextension/ar/akwam/Akwam.kt

```diff
@@ -100,7 +100,7 @@ class Akwam : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
 
     override fun videoListParse(response: Response): List<Video> {
         val document = response.asJsoup()
-        val iframe = "https://akwam.im/watch" + document.select("a.link-show").attr("href").substringAfter("watch") + "/" + document.ownerDocument()!!.select("input#page_id").attr("value")
+        val iframe = "https://akw-cdn1.link/watch" + document.select("a.link-show").attr("href").substringAfter("watch") + "/" + document.ownerDocument()!!.select("input#page_id").attr("value")
```

Comment:
Why not just do `"$baseUrl/watch"` so you don't have to update the link twice when the site changes domain?

Reply:
sometimes i need just change this url as it the site url is different from watch url